data/src/main/java/com/teamagam/gimelgimel/data/geometry/repository/GeoDataRepository.java

```diff
@@ -0,0 +1,72 @@
+package com.teamagam.gimelgimel.data.geometry.repository;
+
+import com.teamagam.gimelgimel.data.geometry.repository.inMemory.InMemoryGeoDataCache;
+import com.teamagam.gimelgimel.domain.geometries.entities.GeoEntity;
+import com.teamagam.gimelgimel.domain.geometries.entities.VectorLayer;
+import com.teamagam.gimelgimel.domain.geometries.repository.GeoRepository;
+
+import rx.Observable;
+
+public class GeoDataRepository implements GeoRepository {
+
+    private InMemoryGeoDataCache mCache;
+
+    public GeoDataRepository(InMemoryGeoDataCache cache) {
+        mCache = cache;
+    }
+
+    @Override
+    public Observable<VectorLayer> getAllVectorLayers() {
```

Comment:
Why?
We can prevents redundant flat mapping when using this method.
An observable should be a sequence of data, and should not contain one list of data.